whatwg.org/chat

```diff
@@ -27,38 +27,42 @@
+<h3 id="chat">Chat<a class="self-link" href="#irc"></a></h3>
 
-<p>The Freenode IRC network has a channel called <a href="ircs://irc.freenode.net/whatwg">#whatwg</a> where some of the more active WHATWG community members hang out. Feel free to join us!</p>
+<p>We have both a <a href="ircs://irc.freenode.net/whatwg">#whatwg</a> channel on the Freenode IRC network and a <a href="https://app.element.io/#/room/#whatwg:matrix.org">whatwg</a> room in the <a href="https://en.wikipedia.org/wiki/Matrix_(protocol)">Matrix</a> network. Join us!</p>
 
-<p>Note that if you go to IRC to ask a question, it might take a while to get a reply. It can pay off to stick around for a couple of hours or more.</p>
+<p>Note that if you ask a question in either of those places, it might take a while to get a reply. It can pay off to stick around for a couple of hours or more.</p>
 
-<p>If you want to run a bot, let us know. If they are useful, e.g. providing logging facilities, then they are more than welcome.</p>
+<h3 id="getting-started">Getting Started with IRC or Matrix<a class="self-link" href="#getting-started"></a></h3>
+
+<p>The simplest way to get started with IRC, if you are not familiar, is by signing up for a free <a href="https://www.irccloud.com/">IRCCloud</a> account. Once you've done that, you should be logged in to the Freenode server by default. All you'll have to do is join the #whatwg channel. If you tell your browser to use IRCCloud for irc:// links, then just <a href="ircs://irc.freenode.net/whatwg">clicking this link</a> should also take you there. In order to join the channel, you will need to <a href="https://freenode.net/kb/answer/registration">register your nick on Freenode</a>.</p>
+
+<p class="note">In view of ongoing spam messaging on Freenode, we highly suggest you <a href="https://freenode.net/kb/answer/usermodes">set your mode</a> to <code>+r</code> in order to block unregistered users from sending you messages.</p>
+
+<p>If you’ve never used Matrix, you can get started by opening the <a href="https://matrix.to/#/#whatwg:matrix.org">whatwg Matrix room launch page</a>, which will prompt you to choose a Matrix client to connect from. Choosing the <a href="https://app.element.io/#/room/#whatwg:matrix.org">Element web app</a> will let you connect to the room right way, directly from your browser. Or you can choose from any of the available <a href="https://matrix.org/clients/">Matrix desktop clients and native mobile apps</a>.</p>
 
 <h3 id="logs">Logs<a class="self-link" href="#logs"></a></h3>
 
-<p>Logs for the #whatwg channel can be found here:</p>
+<p>Chat logs can be found in the following places:</p>
 
 <ul>
- <li><a href="https://freenode.logbot.info/whatwg">https://freenode.logbot.info/whatwg</a></li>
- <li><a href="https://krijnhoetmer.nl/irc-logs/">https://krijnhoetmer.nl/irc-logs/</a> (not updated since 2016-04)</li>
+ <li><a href="https://freenode.logbot.info/whatwg">https://freenode.logbot.info/whatwg</a> (IRC channel)</li>
```

Comment:
This is going away in 6 weeks, so it hardly seems worth linking to it.

Reply:
Let's see in 6 weeks. Ideally we'd link to a replacement or a backup before removing this entirely.

Reply:
>> This is going away in 6 weeks, so it hardly seems worth linking to it.
>
> Let's see in 6 weeks. Ideally we'd link to a replacement or a backup before removing this entirely.

For now, I pushed bb25ab3 to show what dropping it would look like. But if we don’t want to drop it, I can just strip out that commit before merging this.

Reply:
I'd rather not drop it until we actually host the archive ourselves somewhere.

Reply:
OK, I’ve stripped out the commit which dropped the https://freenode.logbot.info/whatwg info